README.md

```diff
@@ -48,12 +48,17 @@ Open up your browser to http://localhost:1313 to view the website. As you make c
 
 ## Running the website locally using Hugo
 
-Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
+You can run Hugo directly (not inside a container). You need to install the version of Hugo that is specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
```

Comment:
HI @forvaidya .
I think this change needs to mention that the extended version of Hugo is required.
Hugo publishes more than one type of a given version.
From the theme's documentation:
```
Install a recent release of the Hugo "extended" version (we recommend version 0.53 or later).
```